graylog2-bootstrap/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java

```diff
@@ -74,8 +75,10 @@ public boolean isNoPidFile() {
 
     @Override
     protected void startCommand() {
-        LOG.info("Graylog {} {} starting up (JRE: {}, installed from: {})", commandName, version,
-                Tools.getSystemInformation(), configuration.getInstallationSource());
+        final OS os = OS.getOs();
+
+        LOG.info("Graylog {} {} starting up (JRE: {}, installed from: {}, OS: {})", commandName, version,
```

Comment:
"JVM" instead of "Java Version" would be more concise and I'm somehow wondering if we can find a better term for "Installation source".